lib/routers/establishment/profile/index.js

```diff
@@ -96,4 +96,6 @@ router.put('/:id',
   submit('update')
 );
 
+router.use('/:id/training', require('./training-modules'));
```

Comment:
Should profile actually be under a specific establishment? What happens if you've got two establishments?

Reply:
I think this should be a separate ticket